src/lang-go.ts

```diff
@@ -178,41 +193,64 @@ async function connectAndInitialize(
     root: URL,
     token: string | undefined
 ): Promise<rpc.MessageConnection> {
+    const connectingToGoLangserverHelp = [
+        `Unable to connect to the Go language server at ${address}.`,
+        `Make sure ${'go.address' as keyof Settings} in your Sourcegraph settings is set to the address of the language server (e.g. wss://sourcegraph.example.com/go).`,
+        `Check out the docs at https://github.com/sourcegraph/sourcegraph-go`,
+    ].join('\n')
+
+    const connectingToSourcegraphHelp = [
+        `The Go language server running on ${address} was unable to fetch repository contents from Sourcegraph running on ${sourcegraphURL()}.`,
+        `Make sure ${'go.sourcegraphUrl' as keyof Settings} in your settings is set to the address of Sourcegraph from the perspective of the language server (e.g. http://sourcegraph-frontend:30080 when running in Kubernetes).`,
+        `Check out the docs at https://github.com/sourcegraph/sourcegraph-go`,
```

Comment:
```suggestion
        `Read the full documentation for more information: https://github.com/sourcegraph/sourcegraph-go`,
```

Nit - I'd like this to be a bit less casual :)

Reply:
Accepted 😃